BUILD.bazel

```diff
@@ -17,6 +17,10 @@ COPTS = [
     "-Werror=old-style-cast",
 ]
 
+MSVC_COPTS = [
+    "/std:c++17",
+]
```

Comment:
fyi: If C++17 is the default now, it might make sense to add a .bazelrc file, since I'm also injecting the C++17 standard option by hand in the bindings build.

Reply:
why wouldn't it being in the bazel BUILD files be sufficient?